includes/class-simple-local-avatars.php

```diff
@@ -953,6 +953,7 @@ public function assign_new_user_avatar( $url_or_media_id, $user_id ) {
 		$meta_value['blog_id'] = get_current_blog_id();
 
 		update_user_meta( $user_id, $this->user_key, $meta_value ); // save user information (overwriting old).
+		do_action( 'simple_local_avatar_updated' , $user_id );
```

Comment:
I know we don't strictly follow this currently in this plugin but ideally any hooks we add should have documentation with them, see https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters. Here's an example in this plugin: https://github.com/10up/simple-local-avatars/blob/develop/includes/class-simple-local-avatars.php#L429.

If that could be added to this action and the other one, that would be great